spec/migrations/20170523220843_add_hawkular_id_to_miq_alerts.rb

```diff
@@ -0,0 +1,16 @@
+require_migration
+
+describe AddHawkularIdToMiqAlerts do
```

Comment:
How can I make this work?
Seems invisible to the main app, and there is no rake task in the local to run this spec.